pkgs/applications/audio/cplay-ng/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, python3
+, fetchFromGitHub
+, makeWrapper
+, mpv
+, pulseaudio
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "cplay-ng";
+  version = "5.1.0";
+
+  src = fetchFromGitHub {
+    owner = "xi";
+    repo = "cplay-ng";
+    rev = version;
+    hash = "sha256-pmuZ1NgrQKEJLoJEVC9wp5deFWc6DiX5DpnISeOOw4k=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/cplay-ng \
+      --prefix PATH : ${lib.makeBinPath [ mpv pulseaudio ]}
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/xi/cplay-ng";
+    description = "Simple curses audio player";
+    longDescription = ''
+      cplay is a minimalist music player with a textual user interface written
+      in Python. It aims to provide a power-user-friendly interface with simple
+      filelist and playlist control.
+
+      Instead of building an elaborate database of your music library, cplay
+      allows you to quickly browse the filesystem and enqueue files,
+      directories, and playlists.
+
+      The original cplay was started by Ulf Betlehem in 1998 and is no longer
+      maintained. This is a rewrite that aims to stay true to the original
+      design while evolving with a shifting environment.
+    '';
+    license = licenses.gpl2Plus;
+    platforms = platforms.all;
```

Comment:
```suggestion
```

Python is not supported by "all" platforms.

Reply:
Then python itself is going to block evaluation for this package too. cplay-ng itself does not have any particular platform restriction.

Reply:
I am not understanding your point. Python is the restriction per se.
And, as far as I remember, `meta.platforms` is automatically filled by `buildPythonApplication`. So, there is no need to set it except maybe for restrict it even more.

Reply:
> meta.platforms is automatically filled by buildPythonApplication

oh right. i always forget this. removed.